docs/tools/community.mdx

```diff
@@ -30,17 +38,16 @@ description: 'AgentStack tools from community contributors'
 - [Composio](/tools/tool/composio)
 
 ## Network Protocols
+
 - [Agent Connect](/tools/tool/agent-connect)
 
 ## Application Specific
+
 - [Stripe](/tools/tool/stripe)
 - [Payman](/tools/tool/payman)
+
 <CardGroup cols={1}>
```

Comment:
please undo the formatting changes here :) we have our own formatting rules this competes with